src/NServiceBus.RabbitMQ/Connection/ConfirmsAwareChannel.cs

```diff
@@ -50,7 +52,29 @@ public Task SendMessage(string address, OutgoingMessage message, IBasicPropertie
                 task = TaskEx.CompletedTask;
             }
 
-            routingTopology.Send(channel, address, message, properties);
+            object delayValue;
+            if (properties.Headers.TryGetValue("NServiceBus.Transport.RabbitMQ.Delay", out delayValue))
+            {
+                //TODO: Do we need to remove the header?
+                properties.Headers.Remove("NServiceBus.Transport.RabbitMQ.Delay");
```

Comment:
I remember that we had a discussion with Andreas about headers, and basically we should try to limit leaving headers on the messages if not needed. So :+1: for removing it. But it is more of a preference than a need I think